database/Image.js

```diff
@@ -1,18 +1,24 @@
 const mongoose = require('mongoose');
-const db = require('./myConnection.js');
-mongoose.Promise = global.Promise;
+const db = require('./myConnection.js'); 
 
+mongoose.Promise = global.Promise;
 
 const restaurantSchema = new mongoose.Schema({
-	res_id: Number,
-	images_array: [{
-  	image_id: Number,
-  	image_url: String,
-  	description: String,
-  	taken_by: String
-  }]
+  res_id: Number,
+  images_array: [
+    {
+      image_id: Number,
+      image_url: String,
+      description: String,
+      taken_by: String,
+    },
+  ],
 });
 
-const Restaurent_Gallery = mongoose.model('restaurent', restaurantSchema);
+const RestaurentGallery = mongoose.model('restaurant', restaurantSchema);
+
+let find = (cb) => {
+  RestaurentGallery.find({},cb);
+}
 
-module.exports = Restaurent_Gallery;
\ No newline at end of file
+module.exports = RestaurentGallery;
```

Comment:
You may need to make the find function a function decorator of RestaurentGallery, or find some other way to include find in your module.exports here.